README.rst

```diff
@@ -326,6 +326,8 @@ Apache 2.0 - See `LICENSE`_ for more information.
 
 .. |build| image:: https://travis-ci.org/GoogleCloudPlatform/google-cloud-python.svg?branch=master
    :target: https://travis-ci.org/GoogleCloudPlatform/google-cloud-python
+.. |appveyor| image:: https://ci.appveyor.com/api/projects/status/1akrbcvnw8481fce/branch/master?svg=true
```

Comment:
Big :+1:

I pulled that from https://ci.appveyor.com/project/GoogleCloudPlatform/google-cloud-python/settings/badges and was wondering why it was so opaque